README.MD

```diff
@@ -18,6 +18,10 @@ Clone app into local machine and run the command json-server in order to make su
 
      https://app.swaggerhub.com/apis/chrishamersly/ProductsV2/0.1#trial
 
+  Tuesday - 08.18.202
+    - Setting up express server, working on middleware application, and developing CRUD. 
+
+
 
 
```

Comment:
It also looks like you don't have a master branch, you should be able add a new branch and call it master then set that to default!